HW_visualization.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[210]:
+
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# In[129]:
+
+
+#first task
+#functions for reading gff and bed6 files
+def read_gff(path_to_gff_file):
+    df_gff = pd.read_csv(path_to_gff_file,
+                 sep='\t', 
+                 header=0,
+                 names=['chromosome', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    df_gff = pd.read_csv(path_to_gff_file,
                 sep='\t', 
                 comment="#",
                 names=['chromosome', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes'])
```